README.md

```diff
@@ -334,9 +333,9 @@ Numerics.Comma separated: Int.parser           5390299.000 ns ±   5.52 %
 Numerics.Comma separated: Scanner.scanInt     86885256.500 ns ±   5.40 %         16
 Numerics.Comma separated: String.split       120884645.000 ns ±   2.18 %         11
 Numerics.Double.init                                62.000 ns ± 615.58 %    1000000
-Numerics.Double.parser                              60.000 ns ± 640.87 %    1000000
+Numerics.Double.parser                              92.000 ns ± 249.73 %    1000000
```

Comment:
We were using `strtod_l` and family before, which is what Swift uses under the hood and is just as fast as `Double.init`, but CI was failing to parse due to not being able to get a handle on contiguous memory. May be able to recover this again in the future, but let's be safe for now.